google/cloud/aiplatform/compat/__init__.py

```diff
@@ -21,7 +21,7 @@
 V1BETA1 = "v1beta1"
 V1 = "v1"
 
-DEFAULT_VERSION = V1
+DEFAULT_VERSION = V1BETA1
```

Comment:
This will not be checked in. This will be reverted once upcoming Model Registry GAPIC changes are released from v1beta1 to v1. This PR will not merge until after that occurs.

(Most tests on this PR will fail for that reason too)